styles/typography/typography-preset-7.json

```diff
@@ -112,16 +99,23 @@
 			"button": {
 				"typography": {
 					"fontFamily": "var:preset|font-family|literata",
-					"fontWeight": "600",
-					"letterSpacing": "-0.96px",
+					"fontSize": "var:preset|font-size|medium",
+					"fontWeight": "400",
+					"letterSpacing": "-0.01em",
```

Comment:
Same here, since the letter spacing would be the same as on the body, it does not need to be duplicated.